api/src/YoutubeClient.cpp

```diff
@@ -0,0 +1,159 @@
+#include "YoutubeClient.h"
+
+#include <rapidjson/schema.h>
+#include <rapidjson/stringbuffer.h>
+#include <sstream>
+
```

Comment:
We utilize `std::string` here, so I think we should `#include <string>`

Reply:
it's included int the header